maintainers/maintainer-list.nix

```diff
@@ -11617,6 +11617,12 @@
     githubId = 2389333;
     name = "Andy Tockman";
   };
+  techknowlogick = {
+    email = "techknowlogick@gitea.io";
+    github = "techknowlogick";
+    githubId = 164197;
+    name = "techknowlogick";
+  };
```

Comment:
Please split this into a separate commit in the same PR, so that if the consul commit has to be reverted for some reason, you stay in this list.